### PR TITLE
Docs changes

### PR DESCRIPTION
**Pull request by vitor-faiante**
- Change all mentions of javascript language to js in order to be consistent everywhere across the docs
- Add js code syntax in plugins.md as the code was not encapsulated inside the code syntax
- Change CDN videojs version for the setup.md guide to the latest version of 5.1.0

### CODE REVIEW

**Comment by gkatsev** [comment]
Thanks. Though, unfortunately, the version is out of date again :)

File: docs/guides/setup.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ You can download the Video.js source and host it on your own servers, or use the
 
 ### CDN Version ###
 ```html
-<link href="//vjs.zencdn.net/4.12/video-js.min.css" rel="stylesheet">
-<script src="//vjs.zencdn.net/4.12/video.min.js"></script>
+<link href="//vjs.zencdn.net/5.1.0/video-js.min.css" rel="stylesheet">
```
Comment (gkatsev) [inline docs/guides/setup.md]: Can you add a thing linking to http://videojs.com/getting-started/ saying that you can always get the latest URL there?

**Comment by gkatsev** [comment]
one comment, LGTM otherwise.

**Comment by vitor-faiante** [comment]
@gkatsev Really sorry for the late response I got swamped at work. Do you mean adding the link in the setup.md?

**Comment by gkatsev** [comment]
LGTM. Thanks!

**Pull request closed without merging**